<launch>
    <node name="video_stream_node" pkg="zal_yolov5" type="video_stream_node">
        <param name="camera_index" value="0" />
        <param name="frame_height" value="480" />
        <param name="frame_width" value="640" />
    </node>
</launch>